<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BunnyWindowClass</class>
 <widget class="QMainWindow" name="BunnyWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bunny.exe</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>260</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="RenderArea" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>11</y>
      <width>1161</width>
      <height>831</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listMsg">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDetails">
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>290</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>360</y>
      <width>92</width>
      <height>15</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_text">
       <property name="text">
        <string>Bunnies alive:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blbBunnyCount">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>330</y>
      <width>73</width>
      <height>15</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbl_text_2">
       <property name="text">
        <string>Cycle nr.:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCycles">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>390</y>
      <width>114</width>
      <height>15</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbl_text_3">
       <property name="text">
        <string>Infect Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblInfects">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BunnyWindow.qrc"/>
 </resources>
 <connections/>
</ui>
